table/builder.go

```diff
@@ -152,6 +154,9 @@ func (b *Builder) handleBlock() {
 		copy(b.buf[item.start:], blockBuf)
 		b.bufLock.Unlock()
 
+		// Add the actual size of current block.
+		atomic.AddUint32(&b.actualSize, uint32(len(blockBuf)))
```

Comment:
This code will be executed only when we have encryption or compression enabled. What if it's not enabled? We don't start handle block goroutine